r/tests/testthat/_snaps/dataset-write.md

```diff
@@ -5,39 +5,45 @@
     Error <rlang_error>
       "compression" is not a valid argument for your chosen `format`.
       i You could try using `codec` instead of `compression`.
+      i Supported arguments: "use_legacy_format", "metadata_version", "codec", and "null_fallback"
```

Comment:
@jonkeane This is how it would look like if we choose to print the supported arguments. The only improvement I would make is (once #12179 is merged) use back ticks for variable names.

Reply:
Done